pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -68,7 +68,6 @@ type RowProcessor interface {
 	ProcessRow(context.Context, isql.Txn, roachpb.KeyValue, roachpb.Value) (batchStats, error)
 	GetLastRow() cdcevent.Row
 	SetSyntheticFailurePercent(uint32)
-	ReportMutations(*stats.Refresher)
```

Comment:
was this unintentional?

Reply:
Intentional -- it should have been in https://github.com/cockroachdb/cockroach/pull/134272 really but I was getting it while I was here.